### PR TITLE
🤑 Display recently Paid Channels on the YPP carousel

### PR DESCRIPTION
**Pull request by thesan**
- #4457

### CODE REVIEW

File: packages/atlas/src/api/hooks/channel.ts [hunk 1]
```diff
@@ -81,6 +86,38 @@ export const useBasicChannels = (
   }
 }
 
+type ChannelPayment = { channel: BasicChannelFieldsFragment; amount: BN }
+export const useRecentlyPaidChannels = (): { channels: ChannelPayment[] | undefined; loading: boolean } => {
+  const aMonthAgo = useMemo(() => startOfDay(subMonths(Date.now(), 1)), [])
```
Comment (WRadoslaw) [inline packages/atlas/src/api/hooks/channel.ts]: As discussed

File: packages/atlas/src/providers/assets/assets.provider.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ type OperatorsContextValue = {
   fetchStorageOperators: () => Promise<BagOperatorsMapping>
   userBenchmarkTime: number | null
 }
-const OperatorsContext = createContext<OperatorsContextValue | undefined>(undefined)
+export const OperatorsContext = createContext<OperatorsContextValue | undefined>(undefined)
```
Comment (WRadoslaw) [inline packages/atlas/src/providers/assets/assets.provider.tsx:49]: Why do we need to export it now?

Comment (thesan) [reply packages/atlas/src/providers/assets/assets.provider.tsx:49]: It makes it easier to mock it in the story [here](https://github.com/Joystream/atlas/blob/d5c82c3118790078b91f7403e3366ec50a7c26e4/packages/atlas/src/components/_channel/ChannelCard/PaidChannelCard.stories.tsx#L45). Using the provider requires setting `ApolloProvider` and `QueryClientProvider`, while just passing an empty object value to `OperatorsContext.Provider` is good enough to run the story.

**Comment by thesan** [comment]
Fetching all payment events (493 atm) result in a 133KB transfer. Is this acceptable for Orion for a page that probably get significant traffic ?

image

I set a 2000 limit so it should 4x long term.

**Comment by thesan** [comment]
Trimming the queried fields results in a ~6x transferred data reduction:
image

**Review by attemka — APPROVED** [review]
lgtm. Great job with this request weight reduction, seems much better now

**Pull request merged**